gradle.properties

```diff
@@ -3,18 +3,22 @@ org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
 
 # Minecraft and Forge
-minecraft_version=1.19.1
-forge_version=42.0.0
+minecraft_version=1.19.2
+forge_version=43.0.7
 
 # Version ranges for mods.toml
-minecraft_version_range=[1.19.1]
-loader_version_range=[42,)
-forge_version_range=[42,)
-
-# Mappings https://github.com/noeppi-noeppi/SugarCane/
-# TODO: Back to sugarcane after 1.19 support in parchment is released
+minecraft_version_range=[1.19.2]
+loader_version_range=[43,)
+forge_version_range=[43,)
+
+# Mappings
+# ==== Possible options ====
+# Parchment: mappings_channel=parchment, mappings_version=yyyy.mm.dd
+# Old Parchment: mappings_channel=parchment_previous, mappings_version=yyyy.mm.dd, previous_minecraft_version=1.x.x
+# Official: mappings_channel=official
+# Other: mappings_channel=x, mappings_version=y
 mappings_channel=parchment
```

Comment:
Can we go back to sugarcane?
1.19.2 is also out for these mappings
https://maven.moddingx.org/org/moddingx/sugarcane/sugarcane-1.19.2/2022.08.10/

Reply:
We could if thats in favour, I just personally got bored of switching it on and off

Reply:
we(probably you) only have to do that during version updates which shouldn't happen as often, but I do like sugarcane (I can override it locally just for me, but eh)

Reply:
that's fair, I'll chuck it back in